tests/elis_api_client/test_documents.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import annotations
+
+import pytest
+
+from rossum_api.api_client import Resource
+
+
+@pytest.fixture
+def file_data() -> bytes:
+    with open("tests/data/sample_invoice.pdf", "rb") as f:
+        return f.read()
+
+
+@pytest.mark.asyncio
+class TestDocuments:
+    async def test_retrieve_document_content(self, elis_client, file_data):
+        client, http_client = elis_client
+        http_client.request.return_value = file_data
+
+        document_id = 123
+        result = await client.retrieve_document_content(document_id=document_id)
+
+        http_client.request.assert_called_with(
+            "GET", url=f"{Resource.Document.value}/{document_id}/content"
+        )
+        assert result == file_data
+
+
+class TestDocumentsSync:
+    def test_retrieve_document_content(self, elis_client_sync, file_data):
+        client, http_client = elis_client_sync
+        http_client.request.return_value = file_data
+
+        document_id = 123
+        result = client.retrieve_document_content(document_id=document_id)
+
+        http_client.request.assert_called_with(
+            "GET", url=f"{Resource.Document.value}/{document_id}/content"
+        )
+        assert result == file_data
```

Comment:
Personal nitpick: I would argue that these unit tests are coupled too tightly to the client logic. At the end of the day, they don't really test that the client works, they just test that it's implemented in the way it is implemented. However, we don't have any E2E tests here and I don't know about any better way to unit test an API client (personally I wouldn't do unit tests here at all). So just take it as a non-actionable comment 😄